.github/workflows/pyright.yml

```diff
@@ -13,15 +13,12 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 jobs:
-  linter_name:
-    name: runner / black formatter
+  pyright:
+    name: Type Checking / pyright
```

Comment:
There's also https://github.com/jakebailey/pyright-action which is by a former member of the Pylance team and is now on the TypeScript team (i.e. fellow MS employee).

Reply:
I will take a look. I picked this one because it supports ReviewDog.
![image](https://user-images.githubusercontent.com/9027725/164544029-75b3b807-b936-4fbd-9305-6e9c9a2712ab.png)
Maybe I can wrap his action the same way i did pylint.